cli/base-command.ts

```diff
@@ -263,80 +255,66 @@ export abstract class BaseCommand extends Command {
       tokenProviderOnChain
     );
 
-    if (routerStr == 'legacy') {
-      this._router = new LegacyRouter({
-        chainId,
-        multicall2Provider,
-        poolProvider: new V3PoolProvider(chainId, multicall2Provider),
-        quoteProvider: new OnChainQuoteProvider(
-          chainId,
-          provider,
-          multicall2Provider
-        ),
-        tokenProvider: this.tokenProvider,
-      });
-    } else {
-      const gasPriceCache = new NodeJSCache<GasPrice>(
-        new NodeCache({ stdTTL: 15, useClones: true })
-      );
+    const gasPriceCache = new NodeJSCache<GasPrice>(
+      new NodeCache({ stdTTL: 15, useClones: true })
+    );
 
-      // const useDefaultQuoteProvider =
-      //   chainId != ChainId.ARBITRUM_ONE && chainId != ChainId.ARBITRUM_RINKEBY;
+    // const useDefaultQuoteProvider =
```

Comment:
Can be deleted?

Reply:
I guess

Reply:
https://github.com/violetprotocol/mauve-smart-order-router/pull/32/commits/844226c47e63a5ac8a77945a44478795f32f915d